include/deepworks/image_reader.hpp

```diff
@@ -1,13 +1,16 @@
 #pragma once
 
 #include <string>
+#include <optional>
 
 namespace deepworks {
 
 class Tensor;
 
 namespace io {
 deepworks::Tensor ReadImage(std::string_view);
+
+void ReadImageToTensor(std::string_view path, Tensor&);
```

Comment:
Why don't you call it `ReadImage` as well to decrease number of entities in API for user ?

Reply:
Done